src/sentry/search/snuba/backend.py

```diff
@@ -638,44 +689,117 @@ def snuba_search(
         ):
             continue
         converted_filter = convert_search_filter_to_snuba_query(search_filter)
+        print ("search_filter:", search_filter)
+        print ("Converted filter:", converted_filter)
 
         # Ensure that no user-generated tags that clashes with aggregation_defs is added to having
         if search_filter.key.name in aggregation_defs and not search_filter.key.is_tag:
+            print ("Appending converted filter to having")
             having.append(converted_filter)
         else:
+            print ("Appending converted filter to conditions")
```

Comment:
Are you planning to run the postgres approach and the snuba join approach in parallel to measure the accuracy and the performance difference during the rollout ?